client/styles/components/_sketch-list.scss

```diff
@@ -50,28 +59,125 @@
   font-weight: normal;
 }
 
-.visibility-toggle .sketch-list__trash-button {
-  @extend %hidden-element;
+.sketch-list__dropdown-button {
   width:#{20 / $base-font-size}rem;
   height:#{20 / $base-font-size}rem;
-}
-
-.visibility-toggle:hover .sketch-list__trash-button {
-   @include themify() {
+  transform: translateY(-3px);
+  @include themify() {
     background-color: transparent;
     border: none;
     cursor: pointer;
     padding: 0;
     position: initial;
     left: 0;
     top: 0;
-      & g {
-        opacity: 1;
-        fill: getThemifyVariable('icon-hover-color');
+
+      & polygon {
+        fill: getThemifyVariable('dropdown-color');
       }
   }
 }
 
+.sketch-list__action-dialogue {
+  position: absolute;
+  width: #{162 / $base-font-size}rem;
+  @include themify(){
+    background-color: getThemifyVariable('modal-background-color');
+		box-shadow: 0 0 18px 0 getThemifyVariable('shadow-color');
+  }
+  border-radius: #{6 / $base-font-size}rem;
+  border: 1px solid #0f9dd7;
+  top: 22%;
+  right: 55%;
+  z-index: 100;
+  button {
+    width: 20px;
+  }
+}
+
+.sketch-list__action-label {
+  @include themify(){
+    color: getThemifyVariable('dropdown-color');
+  }
+  padding-left: #{14 / $base-font-size}rem;
+  width: 100%;
+  height: #{36 / $base-font-size}rem;
+  font-family: Montserrat;
+  font-size: #{12 / $base-font-size}rem;
+  font-weight: bold;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: 3;
+  letter-spacing: normal;
+  text-align: left;
+  margin-top: #{2 / $base-font-size}rem;
+  /*&:hover {
+    @include themify(){
+      color: getThemifyVariable('nav-hover-color');
+      path:nth-child(2) {
+        fill: getThemifyVariable('nav-hover-color');
+      }
+    }
+  }*/
```

Comment:
Perhaps this commented code can be removed?

Reply:
Thanks :)